Python2/3 compatible, PEP8 formatting, first test

Ashafix opened the pull request:
* made the code compatible with Python 3 (print function, range vs. xrange)
* made some cosmetic changes to make it more compliant with PEP8
* tried to unify the print statements
* added one basic test to ensure I didn't break anything

charnley commented:
Hey Ashafix, all conversion to pep8 and python3 compatible looks fine, however I don't like the idea of test function inside the script it self. If you create a new .py file and tests the functionality there, then I will merge the pull request.

Ashafix commented:
- added another py2/3 compatibility
- moved the tests to its own file
- renamed `calculate_rmsd` to `calculate_rmsd.py` to make it importable

Had some issues with the email privacy testing, therefore multiple commits :/
